### ISSUE TITLE
Error in translation

### ISSUE THREAD

**Issue by KihongHeo** [facebook/infer#516]
Hi Infer Team,

Infer translates the following code
```c
int arr[10];
int y = sizeof(arr);
```
to 
```
*&y:int=sizeof(unsigned long) [line 3]
```

**Comment by jvillard** [comment]
Thanks for the report(s)! (I think it would be better to open one issue for each problem if you can)

1) Looks like a bug, thanks!
2) Correct, but what is the problem here? Does that lead to incorrect analysis results?
3) Same as 2). Is your C source missing some code, like `z = y`?

**Comment by KihongHeo** [comment]
Hi @jvillard ,

I will open the second one as a new issue.

**Comment by jvillard** [comment]
Should be fixed in fe01f47e418d2030ba701621a34288e6231b72a8

**Comment by KihongHeo** [comment]
That's a good news!
